pkg/etcd/init/init.go

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
thought: if this does ever happen to fail out, having some extra context might be helpful. 

That is, the underlying procedures in most of the functions in this file are just simple etcd requests, so something fails it may be hard to figure out where exactly the error occurred depending on how generic the etcd error returned was.

Something like `fmt.Errorf("while setting up root user: %w", err)` might be helpful.

(non-blocking: I don't have full context on these so feel free to disregard).

Reply:
That's a good idea. I'll wrap the errors in the init functions.